plugin/token/approle/testdata/lookup_token.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This ended up being not used.  Will delete.  Amazing what you find reviewing your own PR :)